<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>BallCutterDialog</class>
 <widget class="QWidget" name="BallCutterDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>538</width>
    <height>658</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>创建球头分区</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout" stretch="1">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QGroupBox" name="groupBox">
       <property name="title">
        <string>详细参数</string>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_2">
        <item>
         <layout class="QGridLayout" name="gridLayout" columnstretch="1,0,0">
          <property name="leftMargin">
           <number>25</number>
          </property>
          <property name="verticalSpacing">
           <number>12</number>
          </property>
          <item row="2" column="1">
           <widget class="QDoubleSpinBox" name="editC">
            <property name="maximum">
             <double>9999.000000000000000</double>
            </property>
            <property name="value">
             <double>42.359999999999999</double>
            </property>
           </widget>
          </item>
          <item row="5" column="2">
           <widget class="QLabel" name="labelUnitD">
            <property name="text">
             <string>°</string>
            </property>
           </widget>
          </item>
          <item row="20" column="2">
           <widget class="QLabel" name="labelUnitI_6">
            <property name="text">
             <string>mm</string>
            </property>
           </widget>
          </item>
          <item row="20" column="0">
           <widget class="QLabel" name="labelGrindingWheelThick">
            <property name="text">
             <string>砂轮厚度</string>
            </property>
           </widget>
          </item>
          <item row="19" column="2">
           <widget class="QLabel" name="labelUnitI_5">
            <property name="text">
             <string>mm</string>
            </property>
           </widget>
          </item>
          <item row="19" column="0">
           <widget class="QLabel" name="labelCylinderChamferB">
            <property name="text">
             <string>倒角长度2</string>
            </property>
           </widget>
          </item>
          <item row="5" column="1">
           <widget class="QDoubleSpinBox" name="editD">
            <property name="minimum">
             <double>-360.000000000000000</double>
            </property>
            <property name="maximum">
             <double>360.000000000000000</double>
            </property>
            <property name="value">
             <double>90.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="1">
           <widget class="QDoubleSpinBox" name="editA">
            <property name="maximum">
             <double>9999.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>1.600000000000000</double>
            </property>
           </widget>
          </item>
          <item row="7" column="1">
           <widget class="QDoubleSpinBox" name="editF">
            <property name="maximum">
             <double>9999.000000000000000</double>
            </property>
            <property name="value">
             <double>5.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="2">
           <widget class="QLabel" name="labelUnitA">
            <property name="text">
             <string>mm</string>
            </property>
           </widget>
          </item>
          <item row="5" column="0">
           <widget class="QLabel" name="LabelD">
            <property name="text">
             <string>D.弧起始角</string>
            </property>
           </widget>
          </item>
          <item row="6" column="1">
           <widget class="QDoubleSpinBox" name="editE">
            <property name="minimum">
             <double>-360.000000000000000</double>
            </property>
            <property name="maximum">
             <double>360.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="0" column="0">
           <widget class="QLabel" name="lable_A">
            <property name="text">
             <string>A.球头直径</string>
            </property>
           </widget>
          </item>
          <item row="6" column="2">
           <widget class="QLabel" name="labelUnitE">
            <property name="text">
             <string>°</string>
            </property>
           </widget>
          </item>
          <item row="2" column="0">
           <widget class="QLabel" name="labelC">
            <property name="text">
             <string>C.砂轮直径</string>
            </property>
           </widget>
          </item>
          <item row="7" column="0">
           <widget class="QLabel" name="labelF">
            <property name="text">
             <string>F.安全高度</string>
            </property>
           </widget>
          </item>
          <item row="2" column="2">
           <widget class="QLabel" name="labelUnitC">
            <property name="text">
             <string>mm</string>
            </property>
           </widget>
          </item>
          <item row="17" column="2">
           <widget class="QLabel" name="labelUnitI_3">
            <property name="text">
             <string>mm</string>
            </property>
           </widget>
          </item>
          <item row="1" column="2">
           <widget class="QLabel" name="labelUnitB">
            <property name="text">
             <string>°</string>
            </property>
           </widget>
          </item>
          <item row="1" column="1">
           <widget class="QDoubleSpinBox" name="editB">
            <property name="minimum">
             <double>-360.000000000000000</double>
            </property>
            <property name="maximum">
             <double>360.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>1.000000000000000</double>
            </property>
            <property name="value">
             <double>18.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="6" column="0">
           <widget class="QLabel" name="labelE">
            <property name="text">
             <string>E.弧终止角</string>
            </property>
           </widget>
          </item>
          <item row="1" column="0">
           <widget class="QLabel" name="labelB">
            <property name="text">
             <string>B.螺旋角</string>
            </property>
           </widget>
          </item>
          <item row="11" column="0">
           <widget class="QLabel" name="labelH">
            <property name="text">
             <string>H.前角</string>
            </property>
           </widget>
          </item>
          <item row="15" column="1">
           <widget class="QSpinBox" name="editL">
            <property name="maximum">
             <number>999999999</number>
            </property>
            <property name="value">
             <number>6</number>
            </property>
           </widget>
          </item>
          <item row="23" column="1">
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
          <item row="14" column="2">
           <widget class="QLabel" name="labelUnitK">
            <property name="text">
             <string>个</string>
            </property>
           </widget>
          </item>
          <item row="9" column="2">
           <widget class="QLabel" name="labelUnitG">
            <property name="text">
             <string>mm</string>
            </property>
           </widget>
          </item>
          <item row="11" column="1">
           <widget class="QDoubleSpinBox" name="editH">
            <property name="maximum">
             <double>360.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="11" column="2">
           <widget class="QLabel" name="labelUnitH">
            <property name="text">
             <string>°</string>
            </property>
           </widget>
          </item>
          <item row="13" column="1">
           <widget class="QSpinBox" name="editJ">
            <property name="maximum">
             <number>99999999</number>
            </property>
            <property name="value">
             <number>2</number>
            </property>
           </widget>
          </item>
          <item row="7" column="2">
           <widget class="QLabel" name="labelUnitF">
            <property name="text">
             <string>mm</string>
            </property>
           </widget>
          </item>
          <item row="13" column="0">
           <widget class="QLabel" name="labelJ">
            <property name="text">
             <string>J.组数</string>
            </property>
           </widget>
          </item>
          <item row="14" column="0">
           <widget class="QLabel" name="labelK">
            <property name="text">
             <string>K.刃数</string>
            </property>
           </widget>
          </item>
          <item row="12" column="1">
           <widget class="QDoubleSpinBox" name="editI">
            <property name="maximum">
             <double>10000000000000000905969664.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>0.050000000000000</double>
            </property>
           </widget>
          </item>
          <item row="13" column="2">
           <widget class="QLabel" name="labelUnitJ">
            <property name="text">
             <string>组</string>
            </property>
           </widget>
          </item>
          <item row="12" column="2">
           <widget class="QLabel" name="labelUnitI">
            <property name="text">
             <string>mm</string>
            </property>
           </widget>
          </item>
          <item row="9" column="0">
           <widget class="QLabel" name="labelG">
            <property name="text">
             <string>G.下切深度</string>
            </property>
           </widget>
          </item>
          <item row="12" column="0">
           <widget class="QLabel" name="labelI">
            <property name="text">
             <string>I.毛坯头部余量</string>
            </property>
           </widget>
          </item>
          <item row="14" column="1">
           <widget class="QSpinBox" name="editK">
            <property name="maximum">
             <number>999999999</number>
            </property>
            <property name="value">
             <number>3</number>
            </property>
           </widget>
          </item>
          <item row="9" column="1">
           <widget class="QDoubleSpinBox" name="editG">
            <property name="maximum">
             <double>100000000000000000000.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.010000000000000</double>
            </property>
            <property name="value">
             <double>0.300000000000000</double>
            </property>
           </widget>
          </item>
          <item row="15" column="0">
           <widget class="QLabel" name="labelL">
            <property name="text">
             <string>L.总刃数</string>
            </property>
           </widget>
          </item>
          <item row="18" column="2">
           <widget class="QLabel" name="labelUnitI_4">
            <property name="text">
             <string>mm</string>
            </property>
           </widget>
          </item>
          <item row="16" column="0">
           <widget class="QLabel" name="labelCylinderDiameter">
            <property name="text">
             <string>圆柱直径</string>
            </property>
           </widget>
          </item>
          <item row="16" column="2">
           <widget class="QLabel" name="labelUnitI_2">
            <property name="text">
             <string>mm</string>
            </property>
           </widget>
          </item>
          <item row="18" column="0">
           <widget class="QLabel" name="labelCylinderChamferA">
            <property name="text">
             <string>倒角长度1</string>
            </property>
           </widget>
          </item>
          <item row="17" column="0">
           <widget class="QLabel" name="labelCylinderLength">
            <property name="text">
             <string>圆柱长度</string>
            </property>
           </widget>
          </item>
          <item row="21" column="0">
           <widget class="QLabel" name="labelGrindingWheelAngle">
            <property name="text">
             <string>砂轮角度</string>
            </property>
           </widget>
          </item>
          <item row="21" column="2">
           <widget class="QLabel" name="labelUnitI_7">
            <property name="text">
             <string>mm</string>
            </property>
           </widget>
          </item>
          <item row="16" column="1">
           <widget class="QDoubleSpinBox" name="editCylinderDiameter">
            <property name="maximum">
             <double>100000000000000.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>0.800000000000000</double>
            </property>
           </widget>
          </item>
          <item row="17" column="1">
           <widget class="QDoubleSpinBox" name="editCylinderLength">
            <property name="maximum">
             <double>999999999999999.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>2.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="18" column="1">
           <widget class="QDoubleSpinBox" name="editCylinderChamferA">
            <property name="maximum">
             <double>100000000000000000.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="19" column="1">
           <widget class="QDoubleSpinBox" name="editCylinderChamferB">
            <property name="maximum">
             <double>999999999999999.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>0.100000000000000</double>
            </property>
           </widget>
          </item>
          <item row="20" column="1">
           <widget class="QDoubleSpinBox" name="editGrindingWheelThick">
            <property name="maximum">
             <double>999999999999.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>4.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="21" column="1">
           <widget class="QDoubleSpinBox" name="editGrindingWheelAngle">
            <property name="maximum">
             <double>360.000000000000000</double>
            </property>
            <property name="singleStep">
             <double>0.100000000000000</double>
            </property>
            <property name="value">
             <double>90.000000000000000</double>
            </property>
           </widget>
          </item>
          <item row="22" column="0">
           <widget class="QCheckBox" name="checkSpine">
            <property name="text">
             <string>切换引导线</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <spacer name="horizontalSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>5</width>
            <height>5</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources>
  <include location="Resources/Custom.qrc"/>
 </resources>
 <connections/>
</ui>
